doc/api/event_loops.rst

```diff
@@ -0,0 +1,93 @@
+.. _event_loops_label:
+
+Event Loops
+===========
+
+.. currentmodule:: simpleline.event_loop
+
+Event loops are the heart of Simpleline. Every event loop is based on the
+:class:`AbstractEventLoop`, and they all work with :ref:`signals <signals_label>`.
+A signal is a message passed to the loop containing some information. Signals are passed to
+an event loop by calling :meth:`AbstractEventLoop.enqueue_signal`. These signals are then
+processed by calling :meth:`AbstractEventLoop.process_signals`. This method can be called by
+an application developer manually or by the :meth:`AbstractEventLoop.run` method, which is
+called by :meth:`App.run() <simpleline.App.run>` to start the Simpleline-based application.
+When a signal is processed, all handlers attached to this signal are called. Signal handler
+assignment is done by the :meth:`AbstractEventLoop.register_signal_handler` method.
+
+Event loops can also be started recursively by :meth:`AbstractEventLoop.execute_new_loop`.
+The old event loop is waiting for this event loop to stop. New loop execution is mandatory for
+modal screens to work, since they can't be interrupted by other screens. This new event
+loop is terminated by closing the last screen in the event loop or by calling
+the :meth:`AbstractEventLoop.close_loop` method.
+
+The last event loop should be terminated by closing the last screen in the screen stack or by
+calling :meth:`AbstractEventLoop.close_loop`. In case a fatal error occurs the
+:meth:`AbstractEventLoop.force_quit` method can be used to immediately kill the loop.
+
+If a reaction on quitting the application (closing the last event loop) is required, the quit
+callback can be used. The quit callback can be set by
+the :meth:`AbstractEventLoop.set_quit_callback` method.
+
+The following event loops are supported by Simpleline, but you can also
+:ref:`Create_your_own_loop_label` :
+
+* :ref:`MainLoop_label`
+* :ref:`GLib_Event_loop_label`
+
+.. _MainLoop_label:
+
+Main Loop
+---------
+
+The main loop is the default event loop for Simpleline projects. It is an event loop created by
+Simpleline by default. The benefit of using :class:`MainLoop <main_loop.MainLoop>` is that it
```

Comment:
`It is an event loop created by Simpleline by default.`

This seems like the sentence before it, but just reworded a little differently. Maybe it can be removed?